provider/server/server_suite_test.go

```diff
@@ -99,6 +100,11 @@ var _ = BeforeSuite(func() {
 		PathSupportedMachineClasses: machineClassesFile.Name(),
 		RootDir:                     filepath.Join(tempDir, "libvirt-provider"),
 		StreamingAddress:            streamingAddress,
+		Servers: app.ServersOptions{
```

Comment:
If we're not conducting any specific integration tests for metrics, is there a need to include this here?

Reply:
options has to be included but it can be disabled.

Reply:
fixed

Reply:
optional isn't pointer so we don't have to defined it here. But i prefer explicit configuration, developer can immediately see how we run app without study code or docs.